internal/controllers/reflector/labels.go

```diff
@@ -81,9 +81,9 @@ func (r *Controller) reflectLabels(ctx context.Context, deployment *appsv1.Deplo
 	}
 
 	if !allUpdated {
```

Comment:
For the future reference:
There is an error group package in go but it can be confusing for handling multiple errors. Also, there is a great package from michelle (terraform father :) ) https://github.com/hashicorp/go-multierror where yo can do something like


```go
var result error
...
result = multierror.Append(result, err1)
...
result = multierror.Append(result, err2)
...
result.ErrorOrNil() // this can simulate `allUpdated`
```

Reply:
I already use multierror so it makes perfect sense to replace this (actually, I even wanted to do it but decided not to mix it into this PR).
Done, wdyt?

Reply:
totally fine by me.